admin-su/pom.xml

```diff
@@ -2,14 +2,13 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
-    <artifactId>tool-base</artifactId>
+    <artifactId>base</artifactId>
     <groupId>org.sakaiproject</groupId>
     <version>13-SNAPSHOT</version>
-    <relativePath>../../pom.xml</relativePath>
   </parent>
   <name>sakai-tool-tool-su</name>
-  <groupId>org.sakaiproject</groupId>
-  <artifactId>sakai-tool-tool-su</artifactId>
+  <groupId>org.sakaiproject.su</groupId>
+  <artifactId>admin-tool-su</artifactId>
```

Comment:
It's generally better if the folder name matches the `artifactId`, I know Sakai is currently bad at this but would be nice if we could start the path to making it better.

https://blog.sonatype.com/2011/01/maven-tip-project-directories-and-artifact-ids/